tests/IonHashTests.ts

```diff
@@ -84,12 +85,19 @@ function writerDigester(writer: Writer, ionData: string | Uint8Array, algorithm:
     hashWriter.digest();
 }
 
+let digestFunctionDigester: Digester = (ionData: string | Uint8Array, algorithm: string, hasherLog: string[]): void => {
```

Comment:
Nit: this line is a bit long.

Reply:
split this line as well as several others of similar length